calcom/synclinear.com
# Add header to get public images. Cut source links.

All thanks to @tommoor for this fix 🙌 

# Summary

- Use the new `public-file-urls-expire-in` header to fetch Linear markdown with public image URLs
- Remove image source links and their swapping logic

## Test Plan

- Open a ticket with an image to the issue markdown via Linear. Ensure the image renders on GitHub.
- Edit the ticket with more images. Ensure new and old images render on GitHub.
- Repeat from GitHub. Ensure the image renders on Linear (there is some flash of "could not load image" before it re-uploads)

## Related Issues

Closes #114
